setup.py

```diff
@@ -78,10 +86,16 @@
     long_description=readme + '\n\n' + history,
     author='REANA',
     author_email='info@reana.io',
-    url='https://github.com/reanahub/reana',
+    url='http://www.reana.io/',
```

Comment:
HTTPS❓

Reply:
We don't have multi-domain certificate yet... coming later.